src/markdown-pages/add-ons/kotsadm.md

```diff
@@ -22,10 +24,26 @@ spec:
     uiBindPort: 8800
     hostname: "hostname"
     applicationNamespace: "kots"
+    disableS3: true
 ```
 
 flags-table
 
+## Side Effects of the `disableS3` flag
+
+Object storage is used by the KOTS addon as default behaviors in the following ways:
+1. Directly, as a way for the console to store support-bundles and previous versions of your application.
+1. Directly, when hostpath or NFS locations are set in the console. The console will dynamically create a Minio instance and attach to the specified source.
+1. Indirectly, as part of the Registry addon, to store container images in airgap deployments.
+1. Indirectly, as part of the Velero addon, to store snapshots in the cluster using the "Internal" option.
+
+When the disableS3 flag is set to `true`, the KOTS admin console will switch to use a PV for all of the direct operations. 
+It will also have the following side effects outside of the KOTS addon:
+1. For new installs, the Regsistry addon behavior will not change. It will use an object store if it is available, otherwise it will use a PV. For upgrades or re-installs, a migration will be performed from an object store to a PV ONLY if the object store is removed from the installer spec.
```

Comment:
```suggestion
1. For new installs, the Registry add-on behavior will not change. It will use an object store if it is available, otherwise it will use a PV. For upgrades or re-installs, a migration will only be performed from an object store to a PV if the object store is removed from the installer spec.
```